scripts/check-events

```diff
@@ -6,12 +6,32 @@ require 'find'
 # These members/tags are common to multiple events
 BYTE_SIZE_COUNT = ['byte_size', 'count']
 
+# SUFFIX => [MESSAGE, COUNTERS, ADDITIONAL_TAGS]
```

Comment:
I think there may have been some miscommunication here. The original version of this had an additional "required" item in this data. In the [component spec](https://github.com/vectordotdev/vector/blob/master/docs/specs/component.md#bytesreceived) these event structs have certain required fields that this was verifying. I think that behavior needs to be restored to ensure compliance with the spec.

Reply:
May I ask you why we'd need to structs to have those fields?
IMO, we need them just to use them as tags in the logs and metrics, and therefore we don't really require them in the structs as long as we have them in the tags.
But if I'm wrong and having those fields in the structs bring a real value, feel free to tell me.

Reply:
I guess you're right, the important detail is that the output logs and counters have the right fields. They can be "virtual" in the struct if they are implied by the context.